hybridse/src/plan/planner.cc

```diff
@@ -348,7 +348,8 @@ base::Status Planner::CreateCreateTablePlan(const node::SqlNode *root, node::Pla
     *output = node_manager_->MakeCreateTablePlanNode(create_tree->GetDbName(), create_tree->GetTableName(),
                                                      create_tree->GetReplicaNum(),
                                                      create_tree->GetPartitionNum(), create_tree->GetColumnDefList(),
-                                                     create_tree->GetDistributionList());
+                                                     create_tree->GetDistributionList(),
```

Comment:
unit tests to be fix:
https://github.com/4paradigm/OpenMLDB/runs/5563834503?check_suite_focus=true#step:4:530